drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -975,25 +975,65 @@ def _get_user_state(self):
     def _get_correct_state(self):
         """
         Returns one of the possible correct states for the configured data.
+
+        Items dropped by user in the correct zones are retained in those zones.
+        If there are more than one correct zone of a group of items which are dropped in
+        wrong zones, they are equally distributed among the correct zones.
```

Comment:
It's hard to understand this part. Something like this could be a bit more user-friendly.

```suggestion
        Incorrect items are equally distributed among their correct zones (if more than one zone is correct).
```